pure.zsh

```diff
@@ -672,6 +673,7 @@ prompt_pure_setup() {
 		git:arrow            cyan
 		git:branch           242
 		git:branch:cached    red
+		git:dirty     		 130
```

Comment:
Do we want to default this to a new color? I'd really like this color to default to whatever `git:branch` color is set to. For example, git:branch is 242, this is 242. Or, user changes git:branch to 130, this is 130. That is, unless the user specifies this one explicitly in which case that overrides it.

Is it too hidden / hard to understand? Thoughts @sindresorhus?

PS. Indentation is a bit off here (mixed tab/spaces).

Reply:
I don't think we want implicit behavior because `git:branch` color changed. They should all be independent. As for the color change, I think it's nice to make the dirty status a little bit more visible than the branch info. I'm not totally sold on the color though, but we can play around with it later on and decide on what color to use.

Reply:
We've decided to go with `218` as the default color.